RouteStepTests.swift

```diff
@@ -1,6 +1,93 @@
 import XCTest
 @testable import MapboxDirections
 
+class RoadTests: XCTestCase {
+    func testEmpty() {
+        let r = Road(name: "", ref: nil, destination: nil, rotaryName: nil)
+        XCTAssertNil(r.names)
+        XCTAssertNil(r.codes)
+        XCTAssertNil(r.destinations)
+        XCTAssertNil(r.destinationCodes)
+        XCTAssertNil(r.rotaryNames)
+    }
+
+    func testNamesCodes() {
+        var r: Road
+
+        // Name only
+        r = Road(name: "Way Name", ref: nil, destination: nil, rotaryName: nil)
+        XCTAssertEqual(r.names!, [ "Way Name" ])
```

Comment:
Force-unwrapping names here will cause the test case to crash on this line if names is nil for some reason. Instead, fall back to an empty array so that this assertion fails but the test can continue and report about other failures.